.github/workflows/soroban-rpc.yml

```diff
@@ -25,19 +29,69 @@ jobs:
       - uses: ./.github/actions/setup-go
         with:
           go-version: ${{ matrix.go }}
-      - run: go test -race -cover -timeout 25m -v ./cmd/soroban-rpc/...
+      - run: |
+          go generate ./cmd/soroban-rpc/...
```

Comment:
We would need to commit the library for every supported architecture, I would like to avoid that but if `go generating` proves hard to maintain, we can implement it.

Reply:
BTW, I have just removed the `go generate` comment and resorted to just use the top makefile (it's cleaner since `go generate` was simply invoking Make). See https://github.com/stellar/soroban-tools/pull/269#discussion_r1088883689